### ISSUE TITLE
99 skill points for Alex Hunter in The Journey

### ISSUE THREAD

**Issue by sergeiwallace** [xAranaktu/FIFA-18---Career-Mode-Cheat-Table#20]
Is this still possible? It was referenced in the README.md but I don't see anywhere to do it?

**Comment by xAranaktu** [comment]
Nope, sorry. I forgot to update "Readme". The journey section has been removed from the table and it will be not added again.
